distribution/changelog.txt

```diff
@@ -31,7 +31,6 @@
 - Change: [#22491] Scrollbars are now hidden if the scrollable widget is not actually overflowing.
 - Change: [#22541] In editor/sandbox mode, tool widgets now appear on the side of the map window, instead of the bottom.
 - Change: [#22592] Cheats have been redistributed along three new tabs: date, staff, and nature/weather.
-- Fix: [#21123] Transparency options are not respected on startup.
```

Comment:
I don't think we should remove this changelog entry from the previous release. I think we should instead list the revert in the changelog for v0.4.15.

Reply:
I think we should, because it list a fix for a problem that never existed in the first place. At the very least, the entry should be changed to make that clear.

Reply:
I'm fine with clarifying the entry, but removing it feels like a step too far, to me.

Either way, the change is considerable enough to warrant listing for the current version.

Reply:
I agree it needs an entry for the current version, just didn’t add one yet. I generally wait until people have tested it.